File: be/SGBD.Domain/Entities/Clienti.cs
namespace SGBD.Domain.Entities
{
    public partial class Clienti : BaseEntity
    {
        public Clienti()
        {
        }
        public string? Nume { get; set; }
        public string? Prenume { get; set; }
        public string? Adresa { get; set; }
        public string? Companie { get; set; }
        public string? Oras { get; set; }
        public string? Cod { get; set; }
        public string? Telefon { get; set; }
    }
}

File: fe/lib/models/clienti.dart
class Clienti {
  double? id;
  String? nume;
  String? prenume;
  String? adresa;
  String? companie;
  String? oras;
  String? cod;
  String? telefon;

  Clienti({
    this.id,
    this.nume,
    this.prenume,
    this.adresa,
    this.companie,
    this.oras,
    this.cod,
    this.telefon,
  });

  factory Clienti.fromJson(Map<String, dynamic> json) {
    return Clienti(
      id: json['id']?.toDouble(),
      nume: json['nume'],
      prenume: json['prenume'],
      adresa: json['adresa'],
      companie: json['companie'],
      oras: json['oras'],
      cod: json['cod'],
      telefon: json['telefon'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'nume': nume,
      'prenume': prenume,
      'adresa': adresa,
      'companie': companie,
      'oras': oras,
      'cod': cod,
      'telefon': telefon,
    };
  }
}
